test/JDBC/expected/temp_tables_oid-vu-verify.out

```diff
@@ -0,0 +1,121 @@
+-- Tables typically consume three OIDs - we checked this in permanent tables.
```

Comment:
nit: please add where the 3 oids are used for

Reply:
done